flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -1097,6 +1097,12 @@ under the License.
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
Why add python dependencies here?

Reply:
if we don't add the python jar as test dependency, `HiveServer2EndpointITCase` and `HiveServer2EndpointStatementITCase` will failed.